README.md

```diff
@@ -1,5 +1,4 @@
 Code Studio Piskel
-======
```

Comment:
This looks like a mistake - we probably still want this to be a heading.  Maybe you meant to change it from underline-style
```md
Code Studio Piskel
======
```
to regular markdown style?
```md
# Code Studio Piskel
```